cockatrice/src/settings/serverssettings.h

```diff
@@ -4,16 +4,23 @@
 #include "settingsmanager.h"
 #include <QObject>
 
+
 class ServersSettings : public SettingsManager
 {
     Q_OBJECT
     friend class SettingsCache;
 
-public:    
+public:
+    QStringList getDefaultHostList() {
+       return QStringList()
```

Comment:
The only reason it's in a function at all and not a constant was because I
couldn't find a way to make it static/const. I took this style from how we
handle the feature set.

On Fri, Mar 4, 2016, 3:04 PM woogerboy21 <notifications@github.com> wrote:

> In cockatrice/src/settings/serverssettings.h
> <https://github.com/Cockatrice/Cockatrice/pull/1841#discussion_r55081506>:
>
> >  class ServersSettings : public SettingsManager
> >  {
> >      Q_OBJECT
> >      friend class SettingsCache;
> >
> > -public:
> > +public:
> > +    QStringList getDefaultHostList() {
> > +       return QStringList()
>
> Is this ok to keep in the header or should the function content be moved
> to the cpp file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cockatrice/Cockatrice/pull/1841/files#r55081506>.
>